src/applications/personalization/profile/components/alerts/SetUp2FAAlert.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
We should either remove this line or have PropTypes for this file

Reply:
oops good catch!